x-pack/plugins/observability_solution/dataset_quality/public/components/dataset_quality/filters/selector.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFilterButton, EuiPopover, EuiPopoverTitle, EuiSelectable, EuiText } from '@elastic/eui';
+import React, { useState } from 'react';
+import type { EuiSelectableOptionCheckedType } from '@elastic/eui/src/components/selectable/selectable_option';
+import { i18n } from '@kbn/i18n';
+
+const selectorLabel = i18n.translate('xpack.datasetQuality.selector.label', {
```

Comment:
TBH, rather than introducing a default value, i would enforce these params by making them mandatory. Reason being, imagine a client using this component for Types, like in your case, forgets to pass NoneAvailable param, then the selector would show `No Options Available` as a fallback, which would not give a perfect experience. Best would be to enforce the client to pass these basic texts so that the client is forced to think how to handle different use cases for their use case.

Reply:
Addressed in [f50a34b](https://github.com/elastic/kibana/pull/190043/commits/f50a34b685f6699fc1cc27be2518ff2bc115fbe0)